presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -54,6 +55,8 @@
     private Duration queryMaxRunTime = new Duration(100, TimeUnit.DAYS);
     private Duration queryMaxCpuTime = new Duration(1_000_000_000, TimeUnit.DAYS);
 
+    private int minWorkerNodesInitialized = 1;
```

Comment:
Looking at the convention in QueryManagerConfig, we rarely have preposition in the config naming. What about query-manager.initial-min-workers-required?